ds-dl-intro/who.md

```diff
@@ -1,3 +1,6 @@
 Learners are expected to have the following knowledge:
 - Basic Python programming skills and familiarity with the Pandas package.
 - Basic knowledge on Machine learning, including the following concepts: Data cleaning, train & test split, type of problems (regression, classification), overfitting & underfitting, metrics (accuracy, recall, etc.).
+
+Note that this workshop is an introduction into deep learning. 
+If you are already familiar with the concepts in the syllabus you might not learn many new things!
```

Comment:
In my opinion, better to say something like (as saying you might not learn many new things sounds negative):
If you are already familiar with the concepts in the syllabus then a more advanced course might be better suited for your interests!

Reply:
Agreed!